server_test.go

```diff
@@ -83,6 +83,10 @@ func TestWithOpenAPIConfig(t *testing.T) {
 	t.Run("with custom values", func(t *testing.T) {
 		s := NewServer(
 			WithOpenAPIConfig(OpenAPIConfig{
+				Server: &openapi3.Server{
+					URL:         "foo",
+					Description: "bar",
+				},
 				SwaggerUrl:       "/api",
 				JsonUrl:          "/api/openapi.json",
```

Comment:
Mmh I prefer our users to code themselves:

```go
s.OpenAPI.Description().Servers = append(...)
```

Rather than giving them a shortcut that does exactly the same. I mean, we have no added value of doing this